packages/homepage/content/articles/templates.md

```diff
@@ -0,0 +1,63 @@
+title: 'Templates'
+description: "Today we're excited to launch a new feature that we believe will speed up your prototyping workflow here in CodeSandbox! Say hello to Templates!"
+date: 07/02/2019
+---
+
+Today we're excited to launch a new feature that we believe will speed up your prototyping workflow here in CodeSandbox! Say hello to templates!
+
+## **What are Templates?**
+
+We've found that for many of our users, after having used one of our built-in Sandbox starters such as React, Vue, or Gatsby for the first time to prototype an application, they often want to create new projects using the same framework with a specific set of dependencies, folder structure, and configuration set up just the way they like it.
+
+This is so common that we decided why not make it easy for users to create their own custom templates which they can quickly bootstrap new projects with! Today's release is a beta of our new Templates system, which we're planning to improve with a bunch of useful new features over time.
+
+In this early beta release, you can quickly turn any existing Sandbox into a Template with just one click. Let's take a look at how simple it is to get started with Templates!
+
+## **How to make your first Template**
+
+Creating a template from an existing sandbox is really easy! From an existing sandbox, pull up the editor and open up the "Project Info" workspace in the upper left corner. If you've just opened one of our official templates, first fork it and make some changes.
+
+![](/images/templates/1.png)
+
+You'll notice there's a new "Make Template" button. Click that and you'll see that your newly created template has been frozen. This is to help prevent making accidental changes to the template whenever you open it from the Dashboard or Create New Sandbox modal. You can toggle this on and off in the Project Info workspace of the template to make edits to the template itself.
+
+![](/images/templates/2.png)
+
+You can also click on the color in the sidebar to change the color you assign to your template. This is to help distinguish it in your Dashboard and in the Create New Sandbox modal.
+
+![](/images/templates/3.png)
+
+You'll also notice that whenever you attempt to save changes to a frozen sandbox, a new modal will pop up asking you whether you would like to unfreeze it for editing or to fork it to create a new sandbox.
+
+![](/images/templates/4.png)
+
+## Using Templates
+
+So now that you've created your first template, how can you use it? Just like the official templates, you can create a new sandbox from one of your templates via the Create New Sandbox modal.
+
+![](/images/templates/5.png)
+
+In the Overview tab, you'll see your most recently used templates up at the top. Switching over to the My Templates tab, you can browse through all of your custom templates.
+
+![](/images/templates/6.png)
+
+You'll also find all of your custom templates within the Dashboard under the My Templates tab.
+
+![](/images/templates/7.png)
```

Comment:
I was thinking it would also be good to mention teams, since it's a quite unknown part of CodeSandbox.

```suggestion
![](/images/templates/7.png)

## Teams

As a team you can have your own templates as well! When you convert a sandbox owned by your team to a template you will be able to see them under "Team Templates". Team templates are very useful for teams that want to have a common starting point for their team.
```